sanic_testing/testing.py

```diff
@@ -109,28 +108,20 @@ async def _local_request(self, method: str, url: str, *args, **kwargs):
                     do_recv = websocket.recv
 
                     async def send(data):
-                        ws_proxy.received.append(data)
+                        ws_proxy.sent.append(data)
                         await do_send(data)
 
                     async def recv():
                         message = await do_recv()
-                        ws_proxy.sent.append(message)
+                        ws_proxy.received.append(message)
```

Comment:
I wanted these to be from the perspective of the server, not the client.

Reply:
ok

Reply:
Only problem is that we can't guaranteed the server received it before putting it to `ws_proxy.received`...

Reply:
True. Which is why I was originally going with the queues.

Reply:
The test client is not meant to make that guarantee. That is for Sanic core to handle.

Reply:
Okay will change it back, but I still feel weird call it received when we don't know whether it was received. 😂

Imagine that someone tries to use this in their CI/Test Automation to test their server's functionality to handle WebSocket messages and only makes assertion against the `received` in the response object (`ws_proxy`). It can be an issue.